src/js/control-bar/control-bar.js

```diff
@@ -68,10 +69,17 @@ ControlBar.prototype.options_ = {
     'descriptionsButton',
     'subsCapsButton',
     'audioTrackButton',
-    'pictureInPictureToggle',
     'fullscreenToggle'
   ]
 };
 
+if ('exitPictureInPicture' in document) {
+  ControlBar.prototype.options_.children.splice(
+    ControlBar.prototype.options_.children.length - 1,
```

Comment:
maybe better to use indexOf here in case the order changes at some point, if you don't agree, then code lgtm

Reply:
This ordering happens so rarely and this is easily noticeable/discoverable.